docs/organizations/using-coding-standards.md

```diff
@@ -4,24 +4,28 @@ Create coding standards on your organization to define and apply shared tool and
 
 Coding standards help you ensure that Codacy analyzes multiple repositories with the same tool and code pattern configurations. For example, you can use a coding standard to ensure that a group of repositories follow the same security rules or coding conventions.
 
-Each repository can only follow one coding standard at a time. Applying a new coding standard to a repository <span class="skip-vale">unassigns</span> any previously applied coding standard.
+!!! note
+    Consider the following when using coding standards:
 
-When you customize the tools or code patterns of a repository that follows a coding standard, Codacy warns you that the repository will stop following the coding standard and asks for your confirmation.
+    -   Applying a coding standard to a repository only affects the configurations of the tools included in the coding standard, while the remaining tool and code pattern configurations remain unchanged.
 
-!!! important
-    Coding standards turn tools with configuration files on and off. Those tool configuration files, however, take precedence over the code patterns defined on the coding standard.
+    -   Each repository can only follow one coding standard at a time. Applying a new coding standard to a repository <span class="skip-vale">unassigns</span> any previously applied coding standard.
 
-## Creating a coding standard {: id="creating"}
+    -   When you customize the tools or code patterns of a repository that follows a coding standard, Codacy warns you that the repository will stop following the coding standard and asks for your confirmation.
 
-!!! note
-    Codacy currently supports up to 10 coding standards per organization.
+    -   Coding standards turn tools with configuration files on and off. Those tool configuration files, however, take precedence over the code patterns defined on the coding standard.
+
+## Creating a coding standard {: id="creating"}
 
 To create a coding standard for your organization:
 
 1.  Open your organization **Coding standards** page, tab **Coding standards**.
 
 1.  Click the button **Create new standard** at the top right-hand corner of the page. This opens a window with the coding standard creation form.
 
+    !!! note
+        Codacy currently supports up to 10 coding standards per organization.
```

Comment:
I like the idea of placing an admonition only when needed in the flow. Some of the ones we're grouping above could also be interspersed like this, for example `Each repository can only follow one coding standard...` under point 6. Thoughts?

Reply:
This is a good suggestion @nicklem, and I'll try to move information to where it's more relevant in a subsequent pull request (see https://github.com/codacy/docs/pull/1735#discussion_r1203871960).